### PR TITLE
brew-mas: rename and edit page

### PR DESCRIPTION
**Pull request by Aracki**
As we discussed in #3020, `brew-mas` needs to be renamed to `mas`. 

I have changed the examples. `mas install` was incorrect and it's fixed now.

### CODE REVIEW

File: pages/osx/mas.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# mas
+
+> Command line interface for the Mac App Store.
+> Homepage: <https://github.com/mas-cli/mas>.
+
+- Show all installed applications and their product identifiers:
+
+`mas list`
+
+- Search for an application with price included:
```
Comment (sbrl) [inline pages/osx/mas.md]: Does this include the price in the search criteria, or does it display the price alongside the results?

File: pages/osx/mas.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Install or update an application:
+
+`mas install {{product_identifier}}`
+
+- Install all pending updates:
+
+`mas upgrade`
+
+- Sign into the Mac App Store for the first time:
```
Comment (sbrl) [inline pages/osx/mas.md]: Perhaps this should be the first example?

File: pages/osx/mas.md (hunk 1 above)
Comment (Aracki) [reply pages/osx/mas.md]: Display price alongside the results.

File: pages/osx/mas.md (hunk 2 above)
Comment (Aracki) [reply pages/osx/mas.md]: It sounds good.

File: pages/osx/mas.md (hunk 1 above)
Comment (sbrl) [reply pages/osx/mas.md]: Right. What about this then?

```suggestion
- Search for an application, displaying the price alongside the results:
```

**Comment by Aracki** [comment]
What should I do regarding the conflict about `brew-mas.md` removed file?

**Pull request merged**